pgpool2-4.6.yaml

```diff
@@ -63,6 +68,45 @@ subpackages:
       runtime:
         - ${{package.name}}
 
+  - name: ${{package.name}}-compat
+    description: "Compatibility package for Upstream pgpool image"
+    dependencies:
+      provides:
+        - pgpool2-compat=${{package.full-version}}
+      runtime:
+        - bash
+        - coreutils
+        - busybox
+        - postgresql
+        - postgresql-client
+        - postgresql-dev
+    pipeline:
+      - name: "Symlink all pgpool binaries to /opt/pgpool-II/bin"
+        runs: |
+          mkdir -p ${{targets.subpkgdir}}/opt/pgpool-II/bin
+          for bin in \
+            pcp_attach_node pcp_detach_node pcp_health_check_stats pcp_node_count \
+            pcp_node_info pcp_pool_status pcp_proc_count pcp_proc_info pcp_promote_node \
+            pcp_recovery_node pcp_stop_pgpool pcp_watchdog_info \
+            pg_enc pg_md5 pgpool pgpool_setup pgproto watchdog_setup wd_cli \
+          ; do
+            ln -s "/usr/bin/${bin}" "${{targets.subpkgdir}}/opt/pgpool-II/bin/${bin}"
+          done
+
+          # copy etc to pgpool/etc
+          mkdir -p "${{targets.contextdir}}/opt/pgpool-II/etc"
+          cp -pr "${{targets.destdir}}/etc/pgpool2/." "${{targets.contextdir}}/opt/pgpool-II/etc/"
+
+          # copy etc to pgpool/etc.default
+          mkdir -p "${{targets.contextdir}}/opt/pgpool-II/etc.default"
+          cp -pr "${{targets.destdir}}/etc/pgpool2/." "${{targets.contextdir}}/opt/pgpool-II/etc.default"
```

Comment:
don't see this in the upstream one, is this required? 

```bash
$ docker run --rm -it --entrypoint=bash pgpool/pgpool
9ae25c3b6074:/$ ls  /opt/pgpool-II/
bin      etc      include  lib      share
```

Reply:
```
docker run --rm -it --entrypoint=bash pgpool/pgpool:4.0.23                                             
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
eda621c0c8d1:/$ ls opt/pgpool-II/
bin/     etc/     include/ lib/     share/

Reply:
Upstream dockerfile:
https://github.com/pgpool/pgpool2_on_k8s/blob/master/pgpool.docker/Dockerfile.pgpool